grpc/applist.proto

```diff
@@ -29,6 +29,7 @@ service ApplicationService {
   rpc GetDeploymentDetail(DeploymentDetailRequest) returns (DeploymentDetailResponse){}
   rpc InstallRelease(InstallReleaseRequest) returns (InstallReleaseResponse){}
   rpc UpgradeReleaseWithChartInfo(InstallReleaseRequest) returns (UpgradeReleaseResponse){}
+  rpc IsReleaseInstalled(ReleaseIdentifier) returns (BooleanResponse){}
```

Comment:
can we name it better ?

Reply:
Any suggestion? I think this name is enough self explanatory.